client/app/bundles/course/admin/admin.js

```diff
@@ -0,0 +1,13 @@
+$(document).ready(() => {
+  $('input.self-directed-learning-checkbox:checkbox').change(
```

Comment:
Should we use `self-directed-learning` or `advance-start-at` here? 

I prefer the latter, as I'd think it is currently better for css to be consistent with backend column names (in the case where we need to do a global search and replace). 

When we move pages to react, that's when we can have different labelling for front-end elements and backend classes?